third_party/move/move-vm/runtime/src/interpreter.rs

```diff
@@ -57,9 +57,21 @@ macro_rules! set_err_info {
 ///
 /// An `Interpreter` instance is a stand alone execution context for a function.
 /// It mimics execution on a single thread, with an call stack and an operand stack.
-pub(crate) struct Interpreter {
+pub(crate) struct Interpreter;
```

Comment:
Where is this used? I see only Impl

Reply:
That's the public API, I didn't want to change the API, the InterpreterImpl will eventually be generic, while Interpreter can stay as is and do the type-erasure.

Reply:
Ah, I see, `impl Interpreter` creates `InterpreterImpl`, makes sense!